backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala

```diff
@@ -290,18 +290,18 @@ trait StandardAsyncExecutionActor extends AsyncBackendJobExecutionActor with Sta
     lazy val environmentVariables = instantiatedCommand.environmentVariables map { case (k, v) => s"""export $k="$v"""" } mkString("", "\n", "\n")
 
     val home = jobDescriptor.taskCall.callable.homeOverride.map { _ (runtimeEnvironment) }.getOrElse("$HOME")
+    val shortId = jobDescriptor.workflowDescriptor.id.shortString
+    // Give the out and error FIFO variables names that are unlikely to conflict with anything the user is doing.
+    val (out, err) = (s"out$shortId", s"err$shortId")
+    val jobShell = configurationDescriptor.globalConfig.as[Option[String]]("system.job-shell").getOrElse("/bin/bash")
```

Comment:
What's a good place for this variable? In the `StandardJobExecutionActorParams` perhaps?

Reply:
No, the `*ActorParams` classes were because I was sick of having to update multiple `Actor` class constructors whenever we wired in a new parameter.

I think it can be yet[[0]](https://github.com/broadinstitute/cromwell/blob/31/backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala#L153) another[[1]](https://github.com/broadinstitute/cromwell/blob/31/backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala#L290) lazy[[2]](https://github.com/broadinstitute/cromwell/blob/31/backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala#L336) val[[3]](https://github.com/broadinstitute/cromwell/blob/31/backend/src/main/scala/cromwell/backend/standard/StandardAsyncExecutionActor.scala#L417) in the `StandardAsyncExecutionActor`.

(Side note, every time I'm back in this code makes we wish we had more time for developer's choice especially after FP book club)